content/methods/discover/cognitive-walkthrough.md

```diff
@@ -1,10 +1,18 @@
 ---
-title: Cognitive Walkthrough
+title: Cognitive walkthrough
 permalink: /methods/discover/cognitive-walkthrough/
-layout: layouts/page
+layout: layouts/method-card
 tags: methods
 eleventyNavigation:
   key: methods
   parent: Discover
   title: Cognitive walkthrough
+method:
+  title: Cognitive walkthrough
+  what: An evaluation method in which people work through a set of representative tasks and ask questions about the task as they go.
+  why: To get quick and early feedback on whether a design solution is easy for a new or infrequent user to learn, and why it is or isn’t easy. This method is useful for catching big issues at any stage in the design process when you don't have access to real users, but it is not a substitute for user evaluation.
+  timeRequired: 30 minutes to one hour per person
+  category: discover
+  last_modified_at: example timestamp
+  content: ./_data/method_content/cognitive-walkthrough.md
```

Comment:
[idea] Can the `./` or even the `./_data/method_content` portion of this be moved to the template where the content is being loaded?  Just thinking it might be easier to not have to worry about the full path here on the method cards.

Reply:
I still don't feel great about having the markdown in a separate file at all, when it could just live here. I'll keep trying to find a better way for this, but will keep this like so in the meantime.